task 4/task 4/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task_4
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int x = Convert.ToInt32(Console.ReadLine());
```

Comment:
Неверный тип данных. 

> Составить программу, которая запрашивает у пользователя **действительное** значение _x_...

```suggestion
            double x = double.Parse(Console.ReadLine());
```